source/cluster-man/doveadm-cluster-site.1.rst.in

```diff
@@ -0,0 +1,89 @@
+.. _man-1_doveadm-cluster-site:
+
+====================
+doveadm-cluster-site
+====================
+
+
+NAME
+====
+
+doveadm-cluster-site - Utility actions for Dovecot cluster site
+
+
+SYNOPSIS
+========
+
+**doveadm** [**-Dv**] [**-f** *formatter*] cluster site add [ **-\-tag** *tag* ] [ **-\-load-balancer** *host* ] *name*
+
+**doveadm** [**-Dv**] [**-f** *formatter*] cluster site remove *name*
+
+**doveadm** [**-Dv**] [**-f** *formatter*] cluster site update [ **-\-status** *online|offline|standby|evacuate* ] [ **-\-load-balancer** *host* ] *-\-self* | *name*
+
+**doveadm** [**-Dv**] [**-f** *formatter*] cluster site status *name*
+
+**doveadm** [**-Dv**] [**-f** *formatter*] cluster site list
+
+
+DESCRIPTION
+===========
+
+**doveadm cluster site** can be used to run actions that interact with and
+modify Dovecot cluster sites.
+
+.. include:: global-options.inc
+
+.. include:: global-options-formatter.inc
+
+
+COMMANDS
+========
+
+
+cluster site add
+----------------
+
+**doveadm** [**-Dv**] [**-f** *formatter*] cluster site add [ **-\-tag** *tag* ] [ **-\-load-balancer** *host* ] *name*
+
+Adds a new site to cluster. Load balancer is the host or IP address where proxy will connect when doing site to site connections.
+New sites are added in online state.
+
+
+cluster site remove
+-------------------
+
+**doveadm** [**-Dv**] [**-f** *formatter*] cluster site remove *name*
+
+Remove a site. Will this cleanup?
```

Comment:
Remove a site. This currently has no safety checks, so be sure to remove first all of the site's backends and remove it from all users' preferred sites.

We should create a Jira task to add at least checks to make sure backends table is empty. And users could be fixed automatically on login, if they aren't already. The cleanups could have another task, since the same issue exists for also backends.

Reply:
please resolve once ticket made.